<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeRangeWidget</class>
 <widget class="QWidget" name="DateTimeRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QDateTimeEdit" name="dateTime1"/>
   </item>
   <item row="0" column="1">
    <widget class="QToolBar" name="toolBar1"/>
   </item>
   <item row="1" column="0">
    <widget class="QDateTimeEdit" name="dateTime2"/>
   </item>
   <item row="1" column="1">
    <widget class="QToolBar" name="toolBar2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
